test/test_playbooks/domain_vars.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Is there a net benefit of putting this in an extra file?

Reply:
I use them for both hosts, fixtures and tests.
I am not sure if I can add variables that both hosts can use within the playbook.

Reply:
Nice catch.